.gitlab/binary_build/system_probe.yml

```diff
@@ -28,6 +28,9 @@
     - $S3_CP_CMD $SRC_PATH/pkg/ebpf/bytecode/build/offset-guess-debug.o $S3_ARTIFACTS_URI/offset-guess-debug.o.$ARCH
     - $S3_CP_CMD $SRC_PATH/pkg/ebpf/bytecode/build/runtime-security.o $S3_ARTIFACTS_URI/runtime-security.o.$ARCH
     - $S3_CP_CMD $SRC_PATH/pkg/ebpf/bytecode/build/runtime-security-syscall-wrapper.o $S3_ARTIFACTS_URI/runtime-security-syscall-wrapper.o.$ARCH
+    - $S3_CP_CMD $SRC_PATH/pkg/ebpf/bytecode/build/runtime/tracer.c $S3_ARTIFACTS_URI/tracer.c.$ARCH
```

Comment:
I noticed `tracer.c` and `runtime-security.c` were missing from the packaging, when I went to add `conntrack.c`, which is why they are also in this PR.

Reply:
Separated out into separate PR: https://github.com/DataDog/datadog-agent/pull/7508